pretty_midi/containers.py

```diff
@@ -179,23 +179,29 @@ def __str__(self):
 
 
 class Lyric(object):
-    """Timestamped lyric text.
+    """Timestamped lyric text with corresponding note.
 
     Attributes
     ----------
     text : str
         The text of the lyric.
     time : float
         The time in seconds of the lyric.
+    note: :class:`pretty_midi.Note` object
+        The note that corresponds to lyric, None for empty text.
```

Comment:
Not sure what you mean by "empty text" here?

Reply:
I meant whitespace (such as '\r'), used for lyrics formatting.